dbt/include/spark/macros/adapters.sql

```diff
@@ -162,3 +162,16 @@
 {% macro spark__generate_database_name(custom_database_name=none, node=none) -%}
   {% do return(None) %}
 {%- endmacro %}
+
+{% macro spark__alter_column_comment(relation, column_dict) %}
+  {% if config.get('file_format', validator=validation.any[basestring]) == 'delta' %}
+    {% for column_name in column_dict %}
+      {% set comment = column_dict[column_name]['description'] %}
+      {% set comment_query %}
+        ALTER TABLE {{ relation }} ALTER COLUMN {{ adapter.quote(column_name) if column_dict[column_name]['quote'] else column_name }} COMMENT '{{ comment }}';
```

Comment:
1. Per comments in #84, didn't you find that `CHANGE COLUMN` was more effective than `ALTER COLUMN` (even though it doesn't seem there should be a difference)?
2. Style nit: We prefer to lowercase keywords in DDL/DML statements